src/pages/change-how-you-travel/index.js

```diff
@@ -105,31 +105,38 @@ const ChangeHowYouTravelPage = () => (
               Styles.howList
             }`}
           >
+            
             <li>
-              <b>
-                88% of travel emissions are from vehicles like cars, lorries and
-                motorbikes
-              </b>
-              . Their assembly process requires energy generating a carbon
-              footprint (yes, electric too). However most of the damage occurs
-              when a fossil fuel burning vehicle hits the road releasing tonnes
+              <b>15% of of all global green-house gas emissions come directly from transportation</b>, for our commute to work, flying for vacation, taking 
+              a train for a business meeting, etc.
+            </li>
+            <li>
+              On top of that 15% is the assembly process, which requires energy, generating 
+              huge carbon emissions for all modes of transport before they're even used! 
+              When a fossil-fuel burning vehicle hits the road it releases tonnes
               of carbon in the process. There are over 1 billion of these
               vehicles on the road today.
             </li>
             <li>
-              <b>11% comes from air travel</b>. The problems are much the same,
-              planes also use fossil fuel emitting greenhouse gases into the
-              atmosphere.
+              <b>2% of the worlds emissions come from air travel</b>, but only 10% of people 
+              in the world fly. 2% is the same amount as processing the entire worlds sewage, 
+              and 100% of the world are involved in that.
             </li>
             <li>
-              Fossil fuels first need to be extracted, refined and transported
-              for use which means more harmful emissions.
+              Fossil fuels first need to be extracted, refined, and transported
+              for use, all of which uses a lot of energy. There are leakages and spillages
+              at every step of this process.
             </li>
             <li>
               Electric cars don’t emit any carbon once on the road, however they
-              do require energy which has to come from somewhere. Until we hit
-              100% renewable energy electric car usage will harm the environment
-              too but nowhere near as much.
+              do require energy which has to come from somewhere. An electric car
+              running on electricity generated by coal can create more CO2 than a petrol car.
```

Comment:
This sounds pretty negative and although I agree with the facts, I'm not sure it's particularly helpful. It might be worth acknowledging the emissions to produce a new electric car, but again I'm not sure how helpful it is? If people have to drive, I 100% want them to be in an electric car, but it's just wording all this right.

Reply:
Ah, just seen some more details about this below. Perhaps ignore the above.